.github/workflows/build_test_deploy.yml

```diff
@@ -31,6 +31,16 @@ jobs:
           path: ./*
           key: ${{ github.sha }}
 
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
```

Comment:
Can we use `stable` instead of `nightly`? I usually think of "nightly" as highly experimental and built every day.

Reply:
I  get this error when trying to build with stable:

```the cargo feature `strip` requires a nightly version of Cargo, but this is the `stable` channel```

I'll pin the version for now though.

Reply:
`strip` feature could be replaced by `strip` command: https://github.com/parcel-bundler/source-map/blob/master/.github/workflows/release.yml#L17 and https://github.com/parcel-bundler/source-map/blob/master/.github/workflows/release.yml#L108

Reply:
@Brooooooklyn Thanks!
It will be fixed by https://github.com/swc-project/swc/pull/1950

Reply:
nice, using stable now